1-exercises/F-for-of-loop/exercise.js

```diff
@@ -12,5 +12,19 @@ let tubeStations = [
 ];
 
 
-// TODO Use a for-of loop to capitalise and output each letter in the string seperately.
+// TODO Use a for-of loop to capitalize and output each letter in the string seperately.
 let str = "codeyourfuture";
+
+// TODO 1
+for (let i = 0; i < tubeStations.length; i++) {
+const index = tubeStations[i];
+console.log(index);
+}
+
+// TODO 2
+let capture = ""
+for (let i = 0; i < str.length;  i++) {
+    capture += str[i];
+}
+
+console.log(capture.toUpperCase());
```

Comment:
Not quite right. There is a difference between a `for` loop and a `for-of` loop.
Please review those and try again using a `for-of` loop